cmake/onnxruntime_graph.cmake

```diff
@@ -81,7 +81,9 @@ endif()
 onnxruntime_add_static_library(onnxruntime_graph ${onnxruntime_graph_lib_src})
 add_dependencies(onnxruntime_graph onnx_proto flatbuffers)
 onnxruntime_add_include_to_target(onnxruntime_graph onnxruntime_common onnx onnx_proto ${PROTOBUF_LIB} flatbuffers)
-
+if(NOT MSVC)
+  target_compile_options(onnxruntime_graph PRIVATE "-Wno-parentheses")
```

Comment:
curious: why not for windows? alternatively do we need an if?

Reply:
This compile flag is GCC/Clang only.  Previously we disable this warning globally. Now I scope it to only where ONNX schemas are defined. onnxruntime_graph  contains schemas.  Ideally we should fix all the warnings and remove such flags.